src/System.Management.Automation/utils/PInvokeDllNames.cs

```diff
@@ -135,6 +135,7 @@ internal static class PinvokeDllNames
         internal const string CreateToolhelp32SnapshotDllName = "api-ms-win-core-toolhelp-l1-1-0";           /*120*/
         internal const string Process32FirstDllName = "api-ms-win-core-toolhelp-l1-1-0";                     /*121*/
         internal const string Process32NextDllName = "api-ms-win-core-toolhelp-l1-1-0";                      /*122*/
+        internal const string GetACPDllName = "api-ms-win-core-localization-l1-2-0.dll";                     /*123*/
```

Comment:
Now I don't run still  dumpbin.exe /exports
MSDN docs say about:
- Nano api-ms-win-core-localization-l1-2-**2**.dll https://msdn.microsoft.com/en-us/library/mt588480%28v=vs.85%29.aspx?#api-ms-win-core-localization-l1-2-2.dll
- Desktop api-ms-win-core-localization-l1-2-**1**.dll https://msdn.microsoft.com/en-us/library/windows/desktop/dn933214(v=vs.85).aspx

What version we should use?